starter-core/src/test/groovy/io/micronaut/starter/feature/messaging/kafka/KafkaStreamsSpec.groovy

```diff
@@ -83,7 +83,7 @@ class KafkaStreamsSpec extends ApplicationContextSpec implements CommandOutputFi
         template.contains('''<artifactId>micronaut-maven-plugin</artifactId>
           <configuration>
             <shared>true</shared>
-          </configuration>''')
```

Comment:
I had to remove this because config with AOT enabled looks like: 

```xml
    <artifactId>micronaut-maven-plugin</artifactId>
          <configuration>
            <shared>true</shared>
            <configFile>aot-${packaging}.properties</configFile>
          </configuration>
```